Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -291,18 +291,24 @@ def condition_(self, fragment, stepList, key, hasHarvest):
 
 class UpgradeWorkflow_trackingMkFit(UpgradeWorkflowTracking):
     def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Digi' in step: stepDict[stepName][k] = merge([self.step2, stepDict[step][k]])
         if 'Reco' in step: stepDict[stepName][k] = merge([self.step3, stepDict[step][k]])
     def condition_(self, fragment, stepList, key, hasHarvest):
         return '2017' in key or '2021' in key
 upgradeWFs['trackingMkFit'] = UpgradeWorkflow_trackingMkFit(
     steps = [
+        'Digi',
+        'DigiTrigger',
         'Reco',
         'RecoGlobal',
     ],
     PU = [],
     suffix = '_trackingMkFit',
     offset = 0.7,
 )
+upgradeWFs['trackingMkFit'].step2 = {
+    '--customise': 'RecoTracker/MkFit/customizeHLTIter0ToMkFit.customizeHLTIter0ToMkFit'
+}
```

Comment:
Do we want to include a customize function for the HLT iter0? (there is a risk for future maintenance effort given that it would be part of a runTheMatrix workflow)